mappings/net/minecraft/block/Block.mapping

```diff
@@ -624,7 +626,9 @@ CLASS net/minecraft/class_197 net/minecraft/block/Block
 		COMMENT @return Whether the block should drop as an item during an explosion
 		ARG 1 explosion
 			COMMENT The explosion that caused the block to break
-	METHOD method_4187 isFire ()Z
+	METHOD method_4187 doImmediateUpdates ()Z
+		COMMENT Returns whether the block's onScheduledTick should be called directly if ServerWorld.immediateUpdates is true, or to schedule it for the next tick.
```

Comment:
Scheduled ticks could be scheduled for several ticks in the future, not necessarily the next tick.

Reply:
I'll change that later today, so please don't merge it yet

Reply:
edited the comment.
I think it can be merged now...